src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationContextFactory.kt

```diff
@@ -49,7 +54,22 @@ class RecallNotificationContextFactory(
     val lastReleasePrisonName = prisonLookupService.getPrisonName(recall.lastReleasePrison!!)
     val sentencingCourtName = courtLookupService.getCourtName(recall.sentencingInfo!!.sentencingCourt)
     val localPoliceForceName = policeForceLookupService.getPoliceForceName(recall.localPoliceForceId!!)
-    return RecallNotificationContext(recall, prisoner, currentUserDetails, currentPrisonName, lastReleasePrisonName, sentencingCourtName, localPoliceForceName)
+    val revocationOrderCreationDate = documentRepository.findByRecallIdAndCategoryAndVersion(
+      recallId.value,
+      DocumentCategory.REVOCATION_ORDER,
+      1
+    )?.createdDateTime
+      ?: OffsetDateTime.now(clock)
```

Comment:
So I'm not certain but do we need this?  
IOWs: should there not always be an actual `revocationOrderCreationDateTime ` at this point?

Reply:
Ah; so if `RevocationOrder` had its own `ContextFactory` then we'd have to be passed that point ... 
So is `revocationOrderCreationDate`  maybe more like `recallNotificationCreationDate` now?